Source/PLATEAURuntime/Public/PLATEAUImportSettings.h

```diff
@@ -116,4 +156,20 @@ class PLATEAURUNTIME_API UPLATEAUImportSettings : public UObject {
 
         return plateau::polygonMesh::MeshGranularity::PerPrimaryFeatureObject;
     }
-};
\ No newline at end of file
+
+    static TMap<int64, FText> GetCategoryNames() {
+        TMap<int64, FText> Items;
+        Items.Add(static_cast<int64>(plateau::dataset::PredefinedCityModelPackage::Building), LOCTEXT("PrimaryFeatureObject", "建築物"));
```

Comment:
LOCTEXTの一つ目の引数はキーなのでそれぞれ別の名称設定お願いします。
```suggestion
        Items.Add(static_cast<int64>(plateau::dataset::PredefinedCityModelPackage::Building), LOCTEXT("Building", "建築物"));
```